packages/ckeditor5-table/src/plaintableoutput.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module table/plaintableoutput
+ */
+
+import { Plugin } from 'ckeditor5/src/core';
+import Table from './table';
+import TableCaption from './tablecaption';
+import TablePropertiesEditing from './tableproperties/tablepropertiesediting';
+
+/**
+ * The plain table output feature.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class PlainTableOutput extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'PlainTableOutput';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ Table, TablePropertiesEditing, TableCaption ];
```

Comment:
Does it really requires those except table? This will enforce those plugins loaded. Is it expected?

Reply:
It turned out they are not required. I just had to add some small amends to handle situations where these are not loaded. I also added some tests to cover this.